### ISSUE TITLE
Invite message is not built correctly

### ISSUE THREAD

**Issue by astorije** [thelounge/lounge#121]
<img width="339" alt="screen shot 2016-02-28 at 02 47 07" src="https://cloud.githubusercontent.com/assets/113730/13378080/a361d400-ddc5-11e5-8d51-6a4f54ac10d8.png">

Lots of PRs and issues about the `invite` command, so I'm not entirely sure which one broke it. Sorry if it's me :-)

Underlying markup:

```html
<div class="msg invite">
	<span class="time">
		02:46
	</span>
	<span class="from"></span>
	<span class="text">
		<a href="#" class="user">jeremie</a>
invited
<a href="#" class="user">lounge-user <span class="inline-channel" role="button" tabindex="0" data-chan="#astorije</a>">#astorije</span></a>
to


	</span>
</div>
```

**Comment by xPaw** [comment]
I believe this is a bug in `slate-irc`.

**Comment by astorije** [comment]
Weird, it indeed depends on the IRC network! On Freenode it works fine, but the previous example was taken on W3C's IRC network: https://www.w3.org/Project/IRC/

I also confirm this bug already exists on `v1.2.1`, so it was not introduced after this release.

**Comment by xPaw** [comment]
Yep, it's a bug in slate-irc's parser. They expect and assume to always have `!` character in the message prefix, which would be there if full hostmask was sent, but I assume W3C's network only sends the name.

**Comment by astorije** [comment]
Fixed by #167.
